test/deploy/linux/nginx/deploy-application/.net/debian/roles/onbeforestart/templates/selfcontained-net5webapplication-nginx-site

```diff
@@ -0,0 +1,13 @@
+server {
+    listen        8080;
```

Comment:
Can we drive that port number from the deploy config value? @gabeobrien feel free to show the specifics